docs/discussions.rst

```diff
@@ -265,11 +292,15 @@ this? Here are a few resources:
   with sync **and** async callers: "Just add await" from Andrew Godwin:
   https://www.youtube.com/watch?v=oMHrDy62kgE
 
+That being said, synchronous defers (see `discussion-sync-defer`) rely on a few
+specific methods that have been duplicated by hand (one async and one sync version).
```

Comment:
@ewjoachim is this comment sill valid? I am wondering about "duplicated by hand".

Reply:
I've clarified it, but it's still valid.